src/Components/Modal/Modal.tsx

```diff
@@ -1,69 +1,66 @@
-import React from "react";
+import React, { useState } from "react";
 import { observer } from "mobx-react";
 import { Button, Container, Col, Row } from "react-bootstrap";
-import styles from "./Modal.module.css";
+import { ErrorToast } from "../ErrorToast/ErrorToast";
 import type { MediaPresenter } from "../MediaPresenter";
-
+import styles from "./Modal.module.css";
+import classnames from "classnames";
 type Props = {
   mediaPresenter: MediaPresenter;
 };
 
 export const Modal: React.FC<Props> = observer(({ mediaPresenter }) => {
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
   const openModal = (): void => {
-    var modal: HTMLElement | null = document.getElementById("magicModal");
-    if (modal != null) {
-      modal.style.display = "block";
-    }
+    setModalOpen(true);
   };
   const closeModal = (): void => {
     var modal: HTMLElement | null = document.getElementById("magicModal");
     if (modal != null) {
-      modal.style.display = "none";
+      // modal.style.display = "none";
+      setModalOpen(false);
     }
   };
   const windowCloseModal = (
     event: React.MouseEvent<HTMLDivElement, MouseEvent>
   ): void => {
     var modal: HTMLElement | null = document.getElementById("magicModal");
     if (event.target === modal && modal != null) {
-      modal.style.display = "none";
+      // modal.style.display = "none";
+      setModalOpen(false);
     }
   };
   return (
     <div>
       <Button
         variant="info"
-        className={styles.createDesignBtn}
         onClick={openModal}
+        className={styles.createDesignBtn}
       >
         Create a design
       </Button>
 
       <div
         id="magicModal"
-        className={styles.modal}
+        // className={modalOpen? styles.modalOpen : styles.modalClose }
+        className={classnames(styles.modal, { [styles.open]: modalOpen })}
         onClick={(event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
           windowCloseModal(event);
         }}
       >
         <Container fluid="sm" className={styles.modalContent}>
+          <ErrorToast />
```

Comment:
No need to remove now, but this `ErrorToast` wasn't part of your original PR. Try to keep PRs focused & not introduce new things once the PR has been reviewed